ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-main.yaml

```diff
@@ -133,6 +133,28 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: openshift-e2e-azure
+- always_run: false
```

Comment:
Let's automatically trigger on changes in `pkg/updatestatus`

Reply:
Done.